patterns/team.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Title: Team
+ * Slug: twentytwentyfour/team
+ * Categories: team
+ * Block Types: core/template-part/team
```

Comment:
This only works if we are targeting a specific block, we can remove this line

Reply:
Fixed